app/helpers/administrate/application_helper.rb

```diff
@@ -4,5 +4,17 @@ def render_field(field, locals = {})
       locals.merge!(field: field)
       render locals: locals, partial: field.to_partial_path
     end
+
+    def display_resource_name(resource_name)
+      resource_name.
```

Comment:
I would think that "resource" would refer to the whole model object, but that's just me.